lib/ast-converter.js

```diff
@@ -499,6 +501,31 @@ module.exports = function(ast, extra) {
         }
 
         /**
+         * Gets a TSNode's accessibility level
+         * @returns {string} accessibility "public", "protected", or "private"
+         */
+        function getTSNodeAccessibility() {
```

Comment:
It's a minor point, but I would usually prefer that TSNode be passed into the helper function so that it is a bit more portable.

I'm sure we'll be refactoring some of this code once the project is a bit more mature and things like this will make that process easier.